src/pages/_main.js

```diff
@@ -70,9 +72,10 @@ const Main = ({ city, attendees }) => {
             <Sponsors sponsors={sponsors} />
           </Panel>
         )}
-        {mainOrganizer && mainOrganizer.length > 0 ? (
-          <Panel heading={mainOrganizer.length > 1 ? 'Organizers' : 'Organizer'}>
-            <Organizers organizers={mainOrganizer} />
```

Comment:
This change will affect the rendering of some existing yaml files, but maybe in a good way, e.g., https://github.com/queerjs/website/blob/dc438b301a3621cf2c532651b067d4593abc4c4e/data/online-2020-16.yaml#L19